packages/grafana-toolkit/README.md

```diff
@@ -300,6 +300,23 @@ If your comments include ES2016 code, then SystemJS v0.20.19, which Grafana uses
 
 To fix this error, remove the ES2016 code from your comments.
 
+### I would like to dynamically import modules in my plugin
+
+Create a webpack.config.js with this content (in the root of _your_ plugin)
+
+```ts
+// webpack.config.js
+module.exports.getWebpackConfig = (config, options) => ({
+  ...config,
+  output: {
+    ...config.output,
+    publicPath: 'public/plugins/plugin-id/',
```

Comment:
**Nit:** maybe I would write something like `'public/plugins/<plugin-id>/'` to make it more obvious which part needs to be customised
```suggestion
    publicPath: 'public/plugins/<plugin-id>/',
```